python/cudf/cudf/core/column/struct.py

```diff
@@ -5,9 +5,19 @@
 
 import cudf
 from cudf.core.column import ColumnBase
+from cudf.core.column.methods import ColumnMethodsMixin
+from cudf.utils.dtypes import is_struct_dtype
 
 
 class StructColumn(ColumnBase):
+    """
+    Column that stores dict-like values
+
+    Every colummn has n children, where n is
+    number of entries in each dictionary
+
+    """
```

Comment:
I don't think we should use the terms `dict` or `dictionary` here since Struct columns are normalized, whereas dictionaries are not. Would say something like `Column that stores fields of values. Every column has n children, where n is the number of fields in the Struct Dtype.`